### PR TITLE
Prep DevTools release notes for 2.37.1

### PR DESCRIPTION
**Pull request by elliette**
Prepare DevTools release notes for 2.37.1

Screenshot 2024-07-03 at 1 46 35 PM

### CODE REVIEW

File: src/content/tools/devtools/release-notes/release-notes-2.37.1-src.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# DevTools 2.37.1 release notes
+
+The 2.37.1 release of the Dart and Flutter DevTools
+includes the following changes among other general improvements.
+To learn more about DevTools, check out the
+[DevTools overview](/tools/devtools/overview).
+
+## General updates
+
+* Improve messaging when a screen is unavailable for the platform of the
```
Comment (kenzieschmoll) [inline src/content/tools/devtools/release-notes/release-notes-2.37.1-src.md:10]: can you change the tense of all entries to past tense? e.g. Improved, fixed, etc.

https://github.com/flutter/devtools/blob/master/packages/devtools_app/release_notes/README.md#updating-next_release_notesmd

**Review by kenzieschmoll — APPROVED** [review]
LGTM after tense changes

File: src/content/tools/devtools/release-notes/release-notes-2.37.1-src.md (hunk 1 above)
Comment (elliette) [reply src/content/tools/devtools/release-notes/release-notes-2.37.1-src.md]: Done!

**Review by parlough — APPROVED** [review]
Thank you for adding this!

**Pull request merged**